Multiplayer/Components/MainMenu/MultiplayerPane.cs

```diff
@@ -95,6 +98,7 @@ private void ShowPasswordPopup()
             return;
 
         popup.labelTMPro.text = Locale.SERVER_BROWSER__PASSWORD;
+        popup.GetComponentInChildren<TMP_InputField>().text = Multiplayer.Settings.Password;
```

Comment:
I don't think the password should get auto filled. Thoughts? If it does it should probably be saved automatically on a per-server basis.

Reply:
I'm thinking of this as more of a stepping stone until we have a proper server browser running.

I could separate it into two settings one for "server port" and one for "default remote port" for now, if that would be preferred?

Reply:
I've updated this so when connecting directly by IP it will remember the password.

I agree once the server browser is functional, it should remember per server.